src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -742,3 +742,12 @@ interface sampleCode {
 }
 
 type OnlyStrings<T> = { [ P in keyof T]: T[P] extends string ? P : never }[keyof T & string];
+
+interface indexHistoryCommandResult {
```

Comment:
don't we have type for that on server side?

Reply:
Server side has only this type:
`Raven.Client.ServerWide.IndexHistoryEntry`

Reply:
let's create type on server side then, and use keyof in handler: https://github.com/ravendb/ravendb/blob/v5.1/src/Raven.Server/Documents/Handlers/IndexHandler.cs#L129

no need to create instance of this class in server side code.

Reply:
done